planner/commit.go

```diff
@@ -68,15 +68,15 @@ func (n *dagScanNode) Kind() string {
 
 func (n *dagScanNode) Init() error {
 	if len(n.spans.Value) == 0 {
-		if n.commitSelect.DocKey.HasValue() {
-			key := core.DataStoreKey{}.WithDocKey(n.commitSelect.DocKey.Value())
+		if n.commitSelect.DocID.HasValue() {
+			dsKey := core.DataStoreKey{}.WithDocID(n.commitSelect.DocID.Value())
```

Comment:
todo: Related to comments on `DataStoreKey` having an out-of-date name, and worsened by the fact that a `ipfs/datastore.Key` is an actual thing that we use in our codebase, I have quite a strong preference for us to not call anything by ipfs/datastore.Key `dsKey`.

Please rename this variable :sweat_smile: `docID` would probably be fine here.

Reply:
Cheers. Will rename.

- [ ] rename: `dsKey` -> `docID`

Reply:
it's also not a `docID`. I'm really happy to have changed from `dockey` to `docID` as it removes some of the confusion with a datastore key. This however is not one of them. We us `dsKey` everywhere to represent a `DataStoreKey`. Renaming to `docID` would just flip the confusion from what we had before.

Reply:
> We us dsKey everywhere to represent a DataStoreKey

Ah - you are right :) I thought we only used it for `datastore.Key`s :) Leave as-is then :)

Reply:
Yea while doing the suggestion I realized the reason I changed it was that the type is not a docID, so will keep as dsKey